src/components/DevModePanel/useDevModeKeyTrigger.ts

```diff
@@ -1,10 +1,12 @@
 import { useEffect } from "react";
 
-export const useDevModeKeyTrigger = (callback?: () => void) => {
+export const useDevModeKeyTrigger = (callback?: ({ shift }) => void) => {
```

Comment:
suggestion: `shift` is any type, would be better to give it type

Reply:
@poulch done